modules/apps/journal/journal-web/src/test/java/com/liferay/journal/web/internal/display/context/JournalEditArticleDisplayContextTest.java

```diff
@@ -267,6 +269,42 @@ public JournalFolderLocalService get() {
 		);
 	}
 
+	@Test
+	public void testGetTimeZone() {
+		TimeZone timeZone = Mockito.mock(TimeZone.class);
+
+		String timeZoneId = RandomTestUtil.randomString();
+		String timeZoneName = RandomTestUtil.randomString();
+
+		Mockito.when(
+			timeZone.getID()
+		).thenReturn(
+			timeZoneId
+		);
+
+		Mockito.when(
+			timeZone.getDisplayName(false, TimeZone.SHORT)
+		).thenReturn(
+			timeZoneName
+		);
+
+		Mockito.when(
+			_themeDisplay.getTimeZone()
+		).thenReturn(
+			timeZone
+		);
+		_journalEditArticleDisplayContext =
```

Comment:
a new line might be needed before this line

Reply:
Thx. Done with the SF issues.